charts/csi-powerstore/values.yaml

```diff
@@ -164,7 +164,7 @@ controller:
     replicationPrefix: "replication.storage.dell.com"
 
   # Image for csi-metadata-retriever
-  metadataretriever: dellemc/csi-metadata-retriever:v1.3.0
+  metadataretriever: dellemc/csi-metadata-retriever:v1.4.0
```

Comment:
Is this updated in the values.yaml of container-storage-modules helm chart as well?